### PR TITLE
Sprint 37 TLT-2108 Add integration test script

### PR DESCRIPTION
**Pull request by rascalking**
cut-pasted from the one for 2114, with the post body and response
handling tweaked.  next time we want one of these, i library it up good.

* also rename integration test for 2114, fix typo in it

### CODE REVIEW

File: mailgun/test_scripts/tlt-2108-handle-missing-attachments-integration-test.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+# figure out which server to post to
+env_name = settings.ENV_NAME
+if env_name not in ('dev', 'qa', 'stage'):
+    env_name = 'dev'
+
+# post it
+url = 'https://lti-emailer.%s.tlt.harvard.edu/mailgun/handle_mailing_list_email_route/' % env_name
+resp = requests.post(url, data=post_body)
+if not resp.status_code == 400:
```
Comment (elliottyates) [inline mailgun/test_scripts/tlt-2108-handle-missing-attachments-integration-test.py]: Why not `400`?

Comment (elliottyates) [inline mailgun/test_scripts/tlt-2108-handle-missing-attachments-integration-test.py]: Is that because it's the one expected?

Comment (elliottyates) [inline mailgun/test_scripts/tlt-2108-handle-missing-attachments-integration-test.py]: In which case, could these scripts output something, and/or have brief comments or `print()` statements/stdout logging that would explain what they're testing and what the output should be?

Comment (rascalking) [inline mailgun/test_scripts/tlt-2108-handle-missing-attachments-integration-test.py]: Yup.  As to why I didn't write this as `if resp.status_code != 400:`, that's beyond me.

**Comment by elliottyates** [comment]
:+1:

File: mailgun/test_scripts/tlt-2108-handle-missing-attachments-integration-test.py (hunk 1 above)
Comment (rascalking) [inline mailgun/test_scripts/tlt-2108-handle-missing-attachments-integration-test.py]: And sure, though I was going for the usual unix-y "be quiet unless something fails".

Comment (elliottyates) [inline mailgun/test_scripts/tlt-2108-handle-missing-attachments-integration-test.py]: That makes sense in the context of an automated script.

Comment (rascalking) [inline mailgun/test_scripts/tlt-2108-handle-missing-attachments-integration-test.py]: Not just, though.

Comment (elliottyates) [inline mailgun/test_scripts/tlt-2108-handle-missing-attachments-integration-test.py]: :) That's a matter of preference, I guess. If QA is running these things, my thought is they'd prefer verbose output. As long as we're all on the same page. If they're going to act like a unix command, maybe the comments about how they're used and what to expect could just be code comments. I don't care that much, but what it means to act unix-y isn't something everyone knows about, or even those who do (like me) don't necessarily assume scripts work like that. I'm usually much better at asking others for documentation than writing it myself, it must be added.

Comment (rascalking) [inline mailgun/test_scripts/tlt-2108-handle-missing-attachments-integration-test.py]: Yah, I opted for adding comments in the code, including what to look for in splunk.

**Pull request merged**